coordinator/tests/services/translation/prescription.spec.ts

```diff
@@ -104,4 +105,50 @@ describe("PertinentInformation2", () => {
     expect(result).toContain(`&lt;patientInfo&gt;${contentString1}&lt;/patientInfo&gt;`)
     expect(result).not.toContain(`<patientInfo>${contentString1}</patientInfo>`)
   })
+
+  test("Prescriber Endorsements are translated when present", () => {
+    const medicationRequests = getMedicationRequests(bundle)
+
+    const prescriptionEndorsements = medicationRequests
+      .map(medicationRequest =>
+        getExtensionForUrlOrNull(medicationRequest.extension, "https://fhir.nhs.uk/R4/StructureDefinition/Extension-PrescriptionEndorsement") as fhir.CodeableConceptExtension)
+
+    expect(prescriptionEndorsements.length).toBeGreaterThan(0)
```

Comment:
Following convention from other tests. If we add more examples to the models in the future it shouldn't be a breaking change and therefore not break the test as functionally there could be 0, 1 or more endorsements, but we need at least 1 to prove the test otherwise we'll get a false positive test passing

Reply:
not sure i follow this, if we add more examples to the models in the future that wont affect this test since its only looking at a particular example right? the test can only be 'broken' by changing the example its currently looking at. even then, someone would have to mess with the number of MedicationRequest resources or remove the particular extension on the request - things im not sure anyone would ever do.

Reply:
this example is changing all the time. and if we set expectations here for this single test then we should change all our expectations for all tests e.g. provenance etc... and if we move away from relying on models for our tests then this effort would be wasted